src/main/java9/module-info.java

```diff
@@ -0,0 +1,7 @@
+module com.amihaiemil.eoyaml {
+    requires static commons.beanutils;
+
+    exports com.amihaiemil.eoyaml;
+    exports com.amihaiemil.eoyaml.exceptions;
+    exports com.amihaiemil.eoyaml.extensions;
+}
```

Comment:
@njdoyle Also, one more question about these exports. 

Will it affect the access modifiers of the classes? I.e. Almost all the classes inside ``com.amihaiemil.eoyaml`` are package-protected and should not be visible to the user.

So, even if someone uses this lib as a Module, they should still not see the package-protected classes.

Reply:
If running eoyaml non-modularly (from the classpath), protection is the same as before.

If eoyaml is used as a module (from the modulepath), your protection is maintained and actually slightly strengthened. Before, other jars could provide code that say they are in the `com.amihaiemil.eoyaml` package and gain visibility in to your package-protected classes. With modules, it is not allowed for multiple modules to have classes in the same package; closing the above protection circumvention tactic.